core/drivers/crypto/caam/caam_desc.c

```diff
@@ -80,12 +80,16 @@ void caam_desc_push(uint64_t *in_entry, paddr_t paddr)
 
 paddr_t caam_desc_pop(uint64_t *out_entry)
 {
-#ifdef CFG_CAAM_BIG_ENDIAN
-	return get_be64(out_entry);
+	const uint32_t *a32 = (const uint32_t *)out_entry;
+#ifdef	CFG_CAAM_BIG_ENDIAN
+	return (SHIFT_U64(get_be32((void *)&a32[0]), 32) |
```

Comment:
These `(void *)` casts should not be needed.
You can drop the outer `()` too.

Reply:
Sure, done. Thank you